.github/CC_SERVER_INFO.md

```diff
@@ -0,0 +1,45 @@
+<!--Thank you for your report. Please click submit new issue below.-->
+<!-----------Remember to tick all relevant boxes when done!---------->
+
+<!------------------------------------------------------------------->
+<!------------------------------------------------------------------->
+<!------------------ DO NOT MODIFY THE TEXT BELOW ------------------->
+<!------------------------------------------------------------------->
+<!------------------------------------------------------------------->
+
+
+##### AC HASH/COMMIT:
+
+https://github.com/chromiecraft/azerothcore-wotlk/commit/7c21d567d59153b37978756a5bf02daf6998e665

```

Comment:
I'd now remove this too. This file will be only used by contributors whenever they need to report an issue on the AC bugtracker that has NOT been reported on CC (otherwise the could just copy paste the full markdown of the existin CC issue, as @locus313 mentioned earlier).

So we can keep this just as a reference of which modules and OS we use on CC, so it's not required to be updated often and also we avoid the duplication of the AC hash (which would be unconvenient to be updated every time).

I'd just keep everything below (and including) `##### OPERATING SYSTEM:`

This would also require an update of the `README.md` but I can take care of it

Reply:
done